src/main/definitions/case.ts

```diff
@@ -92,6 +92,10 @@ export interface Case {
   claimantPensionWeeklyContribution?: number;
   reasonableAdjustments?: YesOrNo;
   reasonableAdjustmentsDetail?: string;
+  gender?: string;
```

Comment:
Let's try to keep these names consistent with the ccd definitions file to minimise confusion. Properties should be called 'claimant_sex', 'claimant_gender_identity', 'claimant_title1', 'claimant_title_other'. See here - https://github.com/hmcts/et-ccd-definitions-englandwales/blob/master/definitions/json/ComplexTypes.json